backend/models/playlist.js

```diff
@@ -7,42 +7,59 @@ class Playlist {
     }
 
     /**
-     * @param term is the search term 
+     * @param book is the selected book
      * @param token is access token for Spotify API 
+     * @param instrumental is the level of instrumentalness desired for the recommendations
      * @returns a list of the 5 top playlists using the provided search term
      */
-    static async search(term, token) {
-        let words = term.split(" ")
-        if (words.length>4) {
-            term = words.slice(0, 4).join(" ")
-        }
-        const url = `https://api.spotify.com/v1/search?q=${term}&type=playlist`
-        const response = await axios.get(url, {
+    static async search(book, token, instrumental) {
+        const words = book.title.split(" ");
+        let title = ""
+        words.length > 4 ? (title = words.slice(0, 4).join(" ")) : (title = book.title)
+        //use first 4 words of the title
+
+        const url = `https://api.spotify.com/v1/search?q=${title}&type=playlist&limit=3`
+        const titleResponse = await axios.get(url, {
             headers: {
               'Authorization': `Bearer ${token}`
             }
           });
-          let songs = await this.getSongs(response.data.playlists.items, token)
-          return await this.getRec(songs, token)
+
+        const genreArray = book.category.split(" / ")
+        let genre = ""
+        genreArray[genreArray.length-1]==="General" ? (genre = genreArray[genreArray.length-2]) : (genre = genreArray[genreArray.length-1])
+        genre = genre.replace("& ", "")
+        //get significant phrase from genre
+
+        const genreUrl = `https://api.spotify.com/v1/search?q=${genre}&type=playlist&limit=3`
+        const genreResponse = await axios.get(genreUrl, {
+            headers: {
+            'Authorization': `Bearer ${token}`
+            }
+        });
+        let titleSongs = await this.getSongs(titleResponse.data.playlists.items, token)
+        let genreSongs = await this.getSongs(genreResponse.data.playlists.items, token)
+        let songs = titleSongs.concat(genreSongs)
+        return await this.getRec(songs, token, instrumental)
     }
 
     /**
      * 
-     * @param playlistList is a list of 5 playlists from the function above
```

Comment:
nit: leave out magic numbers from descriptions so that they are timeless and generic.